### PR TITLE
Plugin data for node view

### PR DESCRIPTION
**Pull request by alxngyn**
fixes issue #183

## Changes in this PR.

- [X] Edited views to take average of the available plugin data from DB

## Testing this PR.

1. Do rake.
2. See table headers and tables

### Expected Output.

Example with generated random nodes with random values for DiskSize:
screencapture-pink-workstation-cass-oregonstate-edu-4567-node-2-1482269871167

@osuosl/devs

### CODE REVIEW

File: views/nodes/show.erb [hunk 1]
```diff
@@ -3,25 +3,24 @@
     <a href="/node/<%= @node.id %>/edit">edit</a>
   </h2>
   <p>
-    Project Id: <%= @node.project_id %> <br>
```
Comment (hsolorza) [inline views/nodes/show.erb]: I think that the Project Id is something that should still be displayed in the views

Comment (alxngyn) [inline views/nodes/show.erb]: While i think it's useful for debugging and url forwarding I don't think it's useful for the users.

**Comment by Kennric** [comment]
Also, can you add a last-updated date on the measurements table?

**Comment by alxngyn** [comment]
Updated code from feedback
download 7

File: views/nodes/show.erb (hunk 1 above)
Comment (Kennric) [inline views/nodes/show.erb]: Make this the project name and remove the projects table below. A node only belongs to one project, and we don't need that project's details here - just a name that links back to the project view.

**Pull request merged**